src/services/outliningElementsCollector.ts

```diff
@@ -89,6 +106,69 @@ namespace ts.OutliningElementsCollector {
             return isFunctionBlock(node) && node.parent.kind !== SyntaxKind.ArrowFunction;
         }
 
+        function isRegionStart(start: number, end: number) {

```

Comment:
>isRegionStart [](start = 17, length = 13)

It would appear to be a precondition that `start` is within a comment.  Consider adding a comment to that effect.